kratos/modeler/modeler.h

```diff
@@ -119,6 +119,17 @@ class Modeler
         KRATOS_ERROR << "This modeler CAN NOT be used for node generation." << std::endl;
     }
 
+    /**
+     * @brief This method provides the defaults parameters to avoid conflicts between the different constructors
+     */
+    virtual const Parameters GetDefaultParameters() const
+    {
+        KRATOS_ERROR << "Calling the base Modeler class GetDefaultParameters. Please implement the GetDefaultParameters in your derived modeler class." << std::endl;
+        const Parameters default_parameters = Parameters(R"({})");
+
+        return default_parameters;
```

Comment:
I don't think that you would need this after the ERROR. If you need it you can reduce it to 1 line.

```suggestion
        return Parameters(R"({})");
```

Reply:
Btw. You do not need this in the base class. Deriving it will just make it less efficient.